scene/3d/lightmap_gi.cpp

```diff
@@ -1001,10 +1001,16 @@ LightmapGI::BakeError LightmapGI::bake(Node *p_from_node, String p_image_data_pa
 				lightmapper->add_directional_light(light->get_bake_mode() == Light3D::BAKE_STATIC, -xf.basis.get_column(Vector3::AXIS_Z).normalized(), linear_color, energy, l->get_param(Light3D::PARAM_SIZE), l->get_param(Light3D::PARAM_SHADOW_BLUR));
 			} else if (Object::cast_to<OmniLight3D>(light)) {
 				OmniLight3D *l = Object::cast_to<OmniLight3D>(light);
-				lightmapper->add_omni_light(light->get_bake_mode() == Light3D::BAKE_STATIC, xf.origin, linear_color, energy * (1.0 / (Math_PI * 4.0)), l->get_param(Light3D::PARAM_RANGE), l->get_param(Light3D::PARAM_ATTENUATION), l->get_param(Light3D::PARAM_SIZE), l->get_param(Light3D::PARAM_SHADOW_BLUR));
+				if (GLOBAL_GET("rendering/lights_and_shadows/use_physical_light_units")) {
```

Comment:
Do I read this change correctly and was physical lighting the default before? Or should there be a calculation applied to energy when physical lighting is turned off?

Reply:
There should not be a calculation applied when physical lighting is turned off. I accidently left the unit conversion in when making the original PR. But the unit conversion is only applicable when using physical light units. This bug resulted in OmniLights appearing too dark in baked lighting when PLU is turned off.